command/server/config.go

```diff
@@ -88,6 +88,9 @@ type Config struct {
 
 	// GRPC has the grpc server related settings
 	GRPC *GRPCConfig
+
+	// Prometheus Server
+	Prometheus *PrometheusConfig `hcl:"prometheus,optional"`
```

Comment:
Add this config inside the 'Telemetry' group in Config instead of creating a new one.